pytest_mock.py

```diff
@@ -10,7 +10,7 @@
 version = '1.2'
 
 
-class MockFixture(object):
+class MockFixture:
```

Comment:
It is good btw to forget about old-style things.. ;)
We have tests for catching that hopefully.. :)